gremlin-test/features/filter/Has.feature

```diff
@@ -514,6 +514,33 @@ Feature: Step - has()
       | v[ripple] |
       | v[peter] |
 
+ Scenario: g_V_hasXname_regexXrMarXX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().has("name", TextP.regex("^mar"))
```

Comment:
please add some tests using unicode characters as well to demonstrate that the implementation works for it.

Reply:
Will do